go.mod

```diff
@@ -2,3 +2,4 @@ module github.com/creack/pty
 
 go 1.13
 
+require golang.org/x/sys v0.0.0-20201119102817-f84b799fce68
```

Comment:
Is there a specific thing from this version we rely on?

If possible, it would be better to not pin this and let the developer use whatever version they want if they need it. If there is a specific thing we need from this version, please add a comment.

Most OSs/Arch don't need it at all and pinning it would artificially increase the complexity and size of the dep tree.

One of the goals of the =pty= lib is to be a system lib, which can be dropped in and forgotten about, without impacting the rest.

Reply:
We don't have actual version pinning or requirements, this was automatically done by `go mod tidy`, and to my knowledge, when a module is used in another package, go build will automatically select the most suitable package version if multiple dependent packages are using same package with different versions.